### ISSUE TITLE
Guard against overdrafts

### ISSUE THREAD

**Issue by nkhoit** [lhillmer/OttoBot#18]
Currently there's no safeguard if someone tries to withdraw momocoins higher than what Ottobot has on hand.

**Comment by lhillmer** [comment]
What should the safeguard against this be? I'm not entirely sure what ottobot would do at that point? Just reject any withdrawal that it can't afford?

**Comment by nkhoit** [comment]
yeah, just say "i'm out of money" or something. I haven't read your code to see what would happen. But just make sure their balance doesn't get deduced and ottobot says something sufficiently informative to what's happening.
